frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/children/$childId/information.tsx

```diff
@@ -181,10 +181,16 @@ export async function action({ context: { session }, params, request }: ActionFu
     },
   });
 
+  const ageCategory = getAgeCategoryFromDateString(parsedDataResult.data.dateOfBirth);
+
   if (!parsedDataResult.data.isParent) {
     return redirect(getPathById('$lang+/_public+/apply+/$id+/adult-child/children/$childId/parent-or-guardian', params));
   }
 
+  if (ageCategory === 'adults') {
```

Comment:
Need to check for `seniors` as weel since the rule is 18 years old and up.

```suggestion
  if (ageCategory === 'adults' || ageCategory === 'seniors') {
```